tests/brun/pubkey-for-exp-3.txt

```diff
@@ -1,3 +1,3 @@
 brun -c '(pubkey_for_exp (q . 3))'
-cost = 4248
+cost = 23054
```

Comment:
Hmm, this makes me think - what integer type to we use to hold the cost in the Rust code?

Reply:
we use `u64`